modules/migrating-azure-dns-zones.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// *
+
+[id="migrating-azure-dns-zones_{context}"]
+= Migrating legacy Azure DNS zones
+
+If you installed an {product-title} version 4.2 or 4.3 cluster on Azure that uses installer-provisioned infrastructure, you must migrate the legacy private DNS zone to the new type of private DNS zone.
+
+For more information about private zones in Azure, see link:https://azure.microsoft.com/en-us/updates/announcing-preview-refresh-for-azure-dns-private-zones-2/[Announcing Preview Refresh for Azure DNS Private Zones].
+
+.Prerequisites
+
+* Obtain the {product-title} installation program.
+
+.Procedure
+
+. Determine your legacy zones:
++
+----
+$ ./openshift-install migrate azure privatedns eligible
+
+INFO[0000] Credentials loaded from file "<path_to>/ServicePrincipal.json"
+INFO[0002] legacy zone=<legacy_zone> resourceGroup=<resource_group_name>
+----
++
+Note the names of the legacy zones.
+
+. Migrate a legacy zone:
++
+----
+$ openshift-install migrate azure privatedns migrate \
+  --old-resource-group=<resource_group> \ <1>
+  --new-resource-group=<new_resource_group> \ <2>
+  --zone=example.com \ <3>
+  --virtual-network=<vnet> \ <4>
+  --virtual-network-resource-group=<resource_group>  \ <5>
+  --link <6>
+----
```

Comment:
This is still a moving target. I should have the user doc updated in my next push. For now, the current commands for listing eligible zones and migrating them are "azure-privatedns-eligible" and "azure-privatedns-migrate". I've also combined the --old-resource-group and --new-resource-group options into just --resource-group. Other than that, this looks good.

Reply:
Thanks! I'll make these changes now and check back in after your PR merges. Do you know when that will be? We're trying to get all the 4.4 content merged by EOD Friday for our initial translation drop.

Reply:
My hope is that is is merged by then as well ;-)

Reply:
It appears it has been merged. So the official method for doing migrations is [here](https://github.com/openshift/installer/pull/3343/files#diff-1a58973fd4d2a19922fce799fdcea569).

Reply:
@jhixson74, thanks! I've updated the commands to match your PR.